packages/fx-core/src/common/samples-config.json

```diff
@@ -209,6 +209,16 @@
             "time": "5min to run",
             "configuration": "Ready for debug",
             "suggested": false
+        },
+        {
+            "id": "video-filters-app-sample",
+            "title": "Video Filters App Sample",
```

Comment:
Could you please confirm if it's `Video Filters` rather than `Video Filter`? @chaiyiping

Reply:
As we discussed, let's update the wording to `Video Filter` to keep consistent?